app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import time
+from datetime import datetime
+
+
+def create_file_every_second():
+    while True:
+        time.sleep(1)
```

Comment:
I think it would be better to move this line to the end of `while` loop. No need to wait before first iteration